storage/innobase/xtrabackup/test/t/xbcloud.sh

```diff
@@ -15,6 +15,30 @@
 
 . inc/common.sh
 
+### kill all running queries
+function kill_sleep_queries()
+{
+  run_cmd $MYSQL $MYSQL_ARGS --force --batch  test <<EOF
+  select concat('KILL ',id,';') from information_schema.processlist
+  where user='root' and info like '%SLEEP%' and  time > 1 into outfile '$MYSQLD_TMPDIR/killall.sql';
+  source $MYSQLD_TMPDIR/killall.sql;
+EOF
+  rm -f $MYSQLD_TMPDIR/killall.sql
+}
```

Comment:
I would say we should make this more generic to allow change parameters such as user, info and time or just check if a `kill -9` on the MySQL client PID will stop the query.

Reply:
using existing function from common.sh